src/seedu/addressbook/data/person/Person.java

```diff
@@ -10,9 +10,9 @@
  */
 public class Person implements ReadOnlyPerson {
 
-    private Name name;
-    private Phone phone;
-    private Email email;
+    public Name name;
```

Comment:
try to make it private. in general, start off with private members, then use protected or public only if absolutely necessary.

Reply:
Okay will try to incorporate that into my code next time.